raft/src/test/java/org/apache/kafka/raft/KafkaRaftClientTest.java

```diff
@@ -1473,6 +1531,62 @@ public void testFollowerLogReconciliation() throws Exception {
         assertSentFetchQuorumRecordsRequest(epoch, 2L, lastEpoch);
     }
 
+    @Test
+    public void testMetrics() throws Exception {
```

Comment:
I think most of the default field testings should be moved to a dedicate class like `KafkaRaftMetricsTest`

Reply:
I'm going to add a separate class just for the class itself, while leaving the interaction call coverage still in this class